spec/beaker/hypervisor/docker_spec.rb

```diff
@@ -751,7 +751,9 @@ module Beaker
             'image' => 'foobar',
           })
 
-          expect( dockerfile ).to be =~ /RUN pacman -S --noconfirm openssh/
+          expect( dockerfile ).to be =~ /RUN pacman --sync --refresh --noconfirm archlinux-keyring/
```

Comment:
I think the [match matcher](https://relishapp.com/rspec/rspec-expectations/v/3-9/docs/built-in-matchers/match-matcher) would be clearer:
```suggestion
          expect( dockerfile ).to match(/RUN pacman --sync --refresh --noconfirm archlinux-keyring/)
```